graphql/appointment/fields.ts

```diff
@@ -131,6 +143,15 @@ export class ExtendedFieldsLectureResolver {
         if (appointment.matchId) {
             return await prisma.lecture.count({ where: { matchId: appointment.matchId, isCanceled: false } });
         }
+        if (appointment.pupilScreeningId) {
+            return await prisma.lecture.count({ where: { pupilScreeningId: appointment.pupilScreeningId, isCanceled: false } });
+        }
+        if (appointment.instructorScreeningId) {
+            return await prisma.lecture.count({ where: { instructorScreeningId: appointment.instructorScreeningId, isCanceled: false } });
+        }
+        if (appointment.tutorScreeningId) {
+            return await prisma.lecture.count({ where: { tutorScreeningId: appointment.tutorScreeningId, isCanceled: false } });
+        }
         throw new Error('Cannot determine total of loose appointment');
```

Comment:
I guess return 1; would also work here and be a good default, same for position. Dont think we need to implement it for all appointment types ...